### PR TITLE
feat(docs): act-1448 - added sentry plugin

### PR DESCRIPTION
**Pull request by TrofimovAnton85**
## Description

- Added sentry plugin to record user Interactions (only for prod)
- Added support for metamask SDK

## Links

https://infura.atlassian.net/browse/ACT-1448

### CODE REVIEW

**Review by Vt2049 — APPROVED** [review]
👍

**Review by Vt2049 — APPROVED** [review]

File: src/components/ParserOpenRPC/index.tsx [hunk 1]
```diff
@@ -173,6 +165,8 @@ export default function ParserOpenRPC({ network, method }: ParserProps) {
     closeComplexTypeView();
   };
 
+  console.log("connected_acc", connected, account);
```
Comment (cedricmagne) [inline src/components/ParserOpenRPC/index.tsx:168]: Could you remove console.log

**Review by cedricmagne — CHANGES_REQUESTED** [review]

File: src/components/ParserOpenRPC/index.tsx (hunk 1 above)
Comment (TrofimovAnton85) [reply src/components/ParserOpenRPC/index.tsx]: fixed in last commit!

**Review by aednikanov — APPROVED** [review]

**Review by alexandratran — APPROVED** [review]

File: src/components/ParserOpenRPC/InteractiveBox/templates/BaseInputTemplate.tsx
Comment (cedricmagne) [inline src/components/ParserOpenRPC/InteractiveBox/templates/BaseInputTemplate.tsx:1]: Screenshot 2024-07-17 at 11 51 34

If you can just change : 
```
inputValue >= 1
```
by : 
```
Number(inputValue) >= 1
```

**Review by cedricmagne — APPROVED** [review]

**Pull request merged**